### ISSUE TITLE
Copy for an 'About Glow' Section

### ISSUE THREAD

**Issue by DavidVorick** [glowlabs-org/landing-page#1]
## What is Glow?

Glow is a blockchain native single methodology pay-it-forward carbon credit
standard focused on photovoltaic solar that uses emissionality to measure
marginal carbon impact and uses forward productivity to establish
additionality.

## What does that mean?

Glow is a standard for certifying carbon credits. A carbon credit is a
metaphorical object that represents one ton of carbon dioxide that was
eliminated from the atmosphere. There are many carbon credit standards today,
and some reputable examples include the American Carbon Registry and Gold
Standard.

Glow appraises projects that reduce carbon dioxide emissions, and creates one
Glow Carbon Credit for each ton of carbon dioxide emissions that was prevented
by the project. The Glow Protocol takes responsibility for the accurate
appraisal of emissions reductions and uses its own reputation to vouch for the
quality of the carbon credits that are created by Glow projects.

Most carbon credit standards operate by giving carbon credits to projects that
reduce emissions or sequester carbon. Under this model, carbon credit buyers
are retroactively paying for activity that already happened, giving buyers
little guarantee that their money is actually going to drive future change.

Glow uses a pay-it-forward financing structure that ensures all money spent on
carbon credits gets used to drive future carbon credit production. Carbon
credit producers receive rewards in tokens that have value based on carbon
credits that were sold previously.

Glow is a blockchain-native protocol, which means all of the governance and
carbon credit issuance happens on the Ethereum blockchain. Glow does offer ways
to transfer carbon credits off of the blockchain and onto more traditional
carbon credit marketplaces, which means that users can acquire Glow carbon
credits without needing to use a blockchain, however all Glow carbon credits
start their life as ERC20 tokens.

Glow is a single methodology carbon credit standard, which means that Glow only
certifies exactly one type of carbon credit. Glow certifies carbon credits when
photovoltaic solar panels push dirty energy off of the power grid. A
photovoltaic solar panel is the blueish-purple-panel-with-white-lines style of
solar panel - it's what most people think of when they think "solar".

Glow measures carbon impact using emissionality, a strategy for counting carbon
emissions reductions by observing all of the power plants that are operating on
a power grid, understanding which power plants are shutting down in response to
solar power becoming available, and then measuring the total emissions that
would have happened if those power plants had not shut down.

Measuring emissionality is only possible on power grids that have considerable
public data available about which power plants are operating at what
capacities, and what emissions are associated with each power plant. Glow
therefore can only certify solar farms operating on compliant power grids.
Thankfully, modern regulations in most of the US, most of Europe, most of Asia,
most of Australia, and many more places aside from those require this data to
be made readily available. Glow can therefore operate in many emissions
intensive parts of the world.

One of Glow's major innovations is forward productivity, which requires solar
farms to contribute 100% of their gross electricity revenue towards the
incentive pool that funds the installation of more solar farms.

Forward productivity creates a massive multiple on the money that is
contributed by carbon credit buyers. $1 spent on carbon credits turns into $1
that gets spent on solar panels. Those solar panels may then produce $0.95
worth of electricity, which can then be used to buy $0.95 of new solar panels.
And then those solar panels may produce $0.91 of electricity, and so on. In
total, forward productivity allows the Glow Protocol to turn $1 of carbon
credit purchases into $10-$20 of future solar panels, maximizing the overall
additionality of the Glow protocol.

Additionality is a complicated subject, but the core concern behind
additionality is "how much of a difference did my dollar actually make?". For
example, if you pay someone not to cut down trees in a forest, so they respond
by cutting down trees in a *different* forest, your dollar is actually not
reducing the total amount of deforestation and therefore has very low
additionality.

Many varieties of solar based carbon credits struggle with additionality. The
evidence is in the math that companies use to drive their solar decisions. In
most cases today, a solar company will decide to install a solar farm first,
and then check how many carbon credits they get later. Because the carbon
credits are not a core part of the decision, the carbon credits have low
additionality.

With Glow, solar farms must agree to give up all sources of revenue to
participate. That revenue is then used to fund the creation of even more solar
farms. This ensures that Glow's economics are a fundamental part of the
decision process when setting up a new solar farm for Glow, and that every
dollar spent on carbon credits is driving as much real change as possible -
change that would be happening without the dollar.

**Comment by DavidVorick** [comment]
## What is Glow?

The Glow Protocol is a blockchain native single methodology pay-it-forward
carbon credit standard focused on photovoltaic solar that uses emissionality to
measure marginal carbon impact and uses forward productivity to establish
additionality.

## What does that mean?

The Glow Protocol is a standard for certifying carbon credits. A carbon credit
is a metaphorical object that represents one ton of carbon dioxide that would
otherwise be in the atmosphere. There are many carbon credit standards today,
and some reputable examples include the American Carbon Registry and Gold
Standard.

The Glow Protocol appraises projects that reduce carbon dioxide emissions, and
creates one Glow Carbon Credit for each ton of carbon dioxide that was
successfully prevented from being released into the atmosphere. The Glow
Protocol takes responsibility for the accurate appraisal of emissions
reductions and uses its own reputation to vouch for the quality of the carbon
credits that are created by Glow projects.

Most carbon credit standards operate by giving carbon credits to projects that
reduce emissions or sequester carbon. Under this model, carbon credit buyers
are retroactively paying for activity that already happened, giving buyers
little guarantee that their money will result in additional future change.

The Glow Protocol uses a pay-it-forward financing structure that ensures all
money spent on carbon credits gets used to drive future carbon credit
production. Carbon credits that get sold today increase the value of the
rewards pool for producers that create carbon credits tomorrow.

Glow is a blockchain-native protocol, which means all of the governance and
carbon credit issuance happens on the Ethereum blockchain. Blockchain
technology is a good match for the carbon credit industry, as it can give
strong guarantees about who owns the carbon credits, how many carbon credits
are in existance, and what the current state of each carbon credit is.

Blockchain based carbon credit platforms have previously struggled, because
most carbon credits are created using legacy technology that does not mesh well
with Blockchain. As a fully blockchain native carbon credit, Glow is a powerful
ally to other blockchain protocols such as Flowcarbon and Toucan.

For carbon credit buyers that do not want to interact with the blockchain, Glow
features mechanisms for moving carbon credits off-chain and into the legacy
carbon credit system. This reduces the security and guarantees of the off-chain
credits to that of the legacy carbon credit system, and increases convenience
for carbon credit buyers that are not yet ready to make the transition to
blockchain.

The Glow Protocol is a single methodology carbon credit standard, which means
that Glow only certifies exactly one type of carbon credit. Specifically, the
Glow Protocol certifies carbon credits when newly constructed photovoltaic
solar panels displace unclean energy from modern power grids. Dirty energy on
modern power grids is responsible for more than 30% of global carbon dioxide
emissions. A photovoltaic solar panel is the thing that most people picture
when they think of solar power: it's a blueish-purle panel with a grid of while
lines.

The Glow protocol measures carbon dioxide emissions reduction using a strategy
called emissionality. Modern grids publish a large amount of data about the
composition of the grid, including auction data that determines which power
plants are operational at which times of the day. By observing the auction, it
is possible to measure which power plants are being pushed out by the presence
of solar farms. Modern grids also have published data on the carbon dioxide
emissions of each power plant, making it possible to measure the exact carbon
impact of a solar farm.

Emissionality can only be measured on grids that have a sufficient amount of
public data about the electricity auctions and about the carbon dioxide
emissions of each power plant. For this reason, Glow is only able to certify
carbon credits from solar farms that are placed on modern electrical grids.
Thankfully, this covers almost the entire United States, most of Europe, much
of Asia, and more regions are supported every year.

The Glow protocol features a novel requirement called forward productivity.
Under forward productivity, solar farms must contribute 100% of their gross
electricity revenue towards the incentive pool that funds the installation of
more solar farms.

By enforcing this requirement, the Glow protocol is in a sense buying the solar
farm with carbon credit incentives. Because the solar farm produces valuable
resources, the Glow protocol takes ownership of these resources and uses them
to incentivize the construction of even more solar farms.

In a practical sense, forward productivity means that every dollar spent on
carbon credits drives significantly more impact than it would without forward
productivity. It allows each dollar spent on Glow Carbon Credits to create as
much as $10 of solar panels that would not otherwise exist, making Glow one of
the most cost effective carbon credit programs on the planet.

## Why only solar? What about wind?

There are many sources of renewable energy, and all of them contribute to a
cleaner climate. On a philosophical level, the Glow community is aligned with
all renewable energy sources such as wind, geothermal, hydro, and others. The
Glow community is also philosophically aligned with other carbon related
efforts such as sealing oil wells and direct-air-capture.

However, one of the major challenges facing the carbon credit industry today is
that there are a very large number of methodologies for reducing carbon
emissions: more than can be easily appraised by experts. Historically, this has
created a large amount of buyer confusion, and is more recently creating large
beauracratic overheads as incumbent carbon credit standards attempt to ensure
that every type of project meets appropriate quality standards.

Rather than try to create a beauracratic standard that generalizes well across
many carbon methodologies, Glow is choosing to focus on exactly one methodology
to improve buyer confidence, to improve quality, and to reduce overall
beauracratic overhead.

Out of all the options for carbon methodologies, solar seemed like the best
pick due to a number of features. The first is that solar can be scaled until
it is single-handedly offsetting more than 30% of today's global carbon
emissions. The second is that solar has been getting cheaper at a very
encouraging rate, and is expected to be highly cost effective in the near
future. Finally, solar works well at low scales as well as at large scales,
meaning nearly any homeowner can participate in the Glow protocol, and as a
side benefit the homeowner can reduce their dependency on the grid.

**Comment by DavidVorick** [comment]
## What is Glow?

The Glow Protocol is a blockchain native single methodology pay-it-forward
carbon credit standard focused on photovoltaic solar that uses emissionality to
measure marginal carbon impact and uses forward productivity to establish
additionality.

## What does that mean?

The Glow Protocol is a standard for certifying carbon credits. A carbon credit
is a metaphorical object that represents one ton of carbon dioxide that would
otherwise be in the atmosphere. There are many carbon credit standards today,
and some reputable examples include the American Carbon Registry and Gold
Standard.

The Glow Protocol appraises projects that reduce carbon dioxide emissions, and
creates one Glow Carbon Credit for each ton of carbon dioxide that was
successfully prevented from being released into the atmosphere. The Glow
Protocol takes responsibility for the accurate appraisal of emissions
reductions and uses its own reputation to vouch for the quality of the carbon
credits that are created by Glow projects.

Most carbon credit standards operate by giving carbon credits to projects that
reduce emissions or sequester carbon. Under this model, carbon credit buyers
are retroactively paying for activity that already happened, giving buyers
little guarantee that their money will result in additional future change.

The Glow Protocol uses a pay-it-forward financing structure that ensures all
money spent on carbon credits gets used to drive future carbon credit
production. Carbon credits that get sold today increase the value of the
rewards pool for producers that create carbon credits tomorrow.

Glow is a blockchain-native protocol, which means all of the governance and
carbon credit issuance happens on the Ethereum blockchain. Blockchain
technology is a good match for the carbon credit industry, as it can give
strong guarantees about who owns the carbon credits, how many carbon credits
are in existance, and what the current state of each carbon credit is.

Blockchain based carbon credit platforms have previously struggled, because
most carbon credits are created using legacy technology that does not mesh well
with Blockchain. As a fully blockchain native carbon credit, Glow is a powerful
ally to other blockchain protocols such as Flowcarbon and Toucan.

For carbon credit buyers that do not want to interact with the blockchain, Glow
features mechanisms for moving carbon credits off-chain and into the legacy
carbon credit system. This reduces the security and guarantees of the off-chain
credits to that of the legacy carbon credit system, and increases convenience
for carbon credit buyers that are not yet ready to make the transition to
blockchain.

The Glow Protocol is a single methodology carbon credit standard, which means
that Glow only certifies exactly one type of carbon credit. Specifically, the
Glow Protocol certifies carbon credits when newly constructed photovoltaic
solar panels displace unclean energy from modern power grids. Dirty energy on
modern power grids is responsible for more than 30% of global carbon dioxide
emissions. A photovoltaic solar panel is the thing that most people picture
when they think of solar power: it's a blueish-purle panel with a grid of while
lines.

The Glow protocol measures carbon dioxide emissions reduction using a strategy
called emissionality. Modern grids publish a large amount of data about the
composition of the grid, including auction data that determines which power
plants are operational at which times of the day. By observing the auction, it
is possible to measure which power plants are being pushed out by the presence
of solar farms. Modern grids also have published data on the carbon dioxide
emissions of each power plant, making it possible to measure the exact carbon
impact of a solar farm.

Emissionality can only be measured on grids that have a sufficient amount of
public data about the electricity auctions and about the carbon dioxide
emissions of each power plant. For this reason, Glow is only able to certify
carbon credits from solar farms that are placed on modern electrical grids.
Thankfully, this covers almost the entire United States, most of Europe, much
of Asia, and more regions are supported every year.

The Glow protocol features a novel requirement called forward productivity.
Under forward productivity, solar farms must contribute 100% of their gross
electricity revenue towards the incentive pool that funds the installation of
more solar farms.

By enforcing this requirement, the Glow protocol is in a sense buying the solar
farm with carbon credit incentives. Because the solar farm produces valuable
resources, the Glow protocol takes ownership of these resources and uses them
to incentivize the construction of even more solar farms.

In a practical sense, forward productivity means that every dollar spent on
carbon credits drives significantly more impact than it would without forward
productivity. It allows each dollar spent on Glow Carbon Credits to create as
much as $10 of solar panels that would not otherwise exist, making Glow one of
the most cost effective carbon credit programs on the planet.

In summary, the Glow Protocol is a blockchain native single methodology pay-it-forward
carbon credit standard focused on photovoltaic solar that uses emissionality to
measure marginal carbon impact and uses forward productivity to establish
additionality.

## Why only solar? What about wind?

There are many sources of renewable energy, and all of them contribute to a
cleaner climate. On a philosophical level, the Glow community is aligned with
all renewable energy sources such as wind, geothermal, hydro, and others. The
Glow community is also philosophically aligned with other carbon related
efforts such as sealing oil wells and direct-air-capture.

However, one of the major challenges facing the carbon credit industry today is
that there are a very large number of methodologies for reducing carbon
emissions: more than can be easily appraised by experts. Historically, this has
created a large amount of buyer confusion, and is more recently creating large
beauracratic overheads as incumbent carbon credit standards attempt to ensure
that every type of project meets appropriate quality standards.

Rather than try to create a beauracratic standard that generalizes well across
many carbon methodologies, Glow is choosing to focus on exactly one methodology
to improve buyer confidence, to improve quality, and to reduce overall
beauracratic overhead.

Out of all the options for carbon methodologies, solar seemed like the best
pick due to a number of features. The first is that solar can be scaled until
it is single-handedly offsetting more than 30% of today's global carbon
emissions. The second is that solar has been getting cheaper at a very
encouraging rate, and is expected to be highly cost effective in the near
future. Finally, solar works well at low scales as well as at large scales,
meaning nearly any homeowner can participate in the Glow protocol, and as a
side benefit the homeowner can reduce their dependency on the grid.

**Comment by DavidVorick** [comment]
## What is Glow?

The Glow Protocol is a blockchain native single methodology pay-it-forward
carbon credit standard focused on solar that uses emissionality to measure
marginal carbon impact and uses forward productivity and competition to
establish additionality.

## What does that mean?

The Glow Protocol is a standard for certifying carbon credits. A carbon credit
is a metaphorical object that represents one ton of carbon dioxide that would
otherwise be in the atmosphere. There are many carbon credit standards today,
and some reputable examples include the American Carbon Registry and CORSIA.

The Glow Protocol appraises projects that reduce carbon dioxide emissions, and
issues one Glow Carbon Credit for each metric ton of carbon dioxide that is
successfully prevented from being released into the atmosphere. The Glow
Protocol takes responsibility for the accurate appraisal of emissions
reductions and uses its own reputation to vouch for the quality of the carbon
credits that are created by Glow projects.

Glow is a blockchain-native protocol, which means all of the governance and
carbon credit issuance happens on the Ethereum blockchain. Blockchain
technology is a good match for the carbon credit industry, as it can give
strong guarantees about who owns the carbon credits, how many carbon credits
are in existance, and the current state of each carbon credit.

Many previously existing blockchain based carbon credit platforms struggled
because it is difficult to migrate legacy carbon credits to the blockchain, and
as a result many platforms either had few carbon credits or carbon credits of
very low quality. Glow supports the rest of the carbon credit blockchain
ecosystem by providing a source of scalable, high quality carbon credits that
permissionlessly interact with the rest of the crypto ecosystem.

Furthermore, it is much easier to migrate a blockchain based carbon credit into
the legacy carbon credit ecosystem than it is to go the other way. Carbon
credit buyers that are not yet ready to make the leap into Blockchain have the
ability to buy and retire Glow carbon credits using traditional payment flows.

Most carbon credit standards operate by giving carbon credits to projects that
reduce emissions or sequester carbon. Carbon credit buyers then purchase the
credits, effectively paying retroactively for change that has already occurred
and receiving little to no assurance that their money will be used to
incentivize future change.

The Glow Protocol uses a pay-it-forward financing structure that ensures all
money spent on carbon credits gets used to drive future carbon credit
production. As carbon credits are sold, the money is used to fuel the incentive
pool that pays out to future carbon projects.

This means that Glow is structured differently than traditional carbon credit
standards. Projects on Glow directly receive cash and tokens for producing
carbon credits instead of receiving carbon credits. The carbon credits, once
produced, are sold by the Glow protocol to fund future projects.

The Glow Protocol is a single methodology carbon credit standard, which means
that Glow only certifies exactly one type of carbon credit. Specifically, the
Glow Protocol certifies carbon credits when newly constructed solar farms
displace dirty energy from modern power grids. Dirty energy makes up more than
30% of all global emissions, giving Glow the opportunity to scale to producing
tens of billions of carbon credits per year even though Glow only uses a single
methodology to produce carbon credits.

The choice to use a single methodolgy is intentional. The carbon credit
ecosystem today features over one hundred different methodolgies, each with
their own controversies and criticisms. Most buyers of carbon credits have
little hope of understanding what exactly they are purchasing, and education is
difficult because every carbon credit has its own story, and its own drawbacks.

By focusing on a single methodology, the Glow protocol is better able to scale
up, better able to address shortcomings and criticisms, and is better able to
give carbon credit buyers confidence in the credits they are purchasing.

The Glow protocol measures carbon dioxide emissions reductions using a strategy
called emissionality. Emissionality uses the rich, regulated data feeds of
modern power grids to observe which power plants reduce their output in
response to new solar farms joining the power grid. This allows Glow to measure
the precise emissions impact of a single solar farm, and also means that each
solar farm will produce different amounts of carbon credits based on where they
are in the world.

Emissionality can only be measured on grids that publish high quality data
about which power plants are active and how many emissions are associated with
each power plant. Thankfully, modern regulations require this data to be
available, and thus Glow is able to certify solar in the entire continential
US, most of Europe, much of Asia, and significant portions of the rest of the
world, with more regions being supported every year. Glow sources most of its
data through a non-profit provider called WattTime.

The Glow protocol features a novel requirement called forward productivity,
which requires solar farms to contribute 100% of their gross electricity
revenue towards the incentive pool that rewards solar farms for producing
carbon credits.

Though forward productivity sounds unusual at first, it solves a very important
incentive alignment problem that exists in much of the carbon credit space.
Carbon credit buyers are spending money on carbon credits because they want to
change the world. This means that their money needs to inspire changes that
would not happen otherwise.

If solar farms are allowed to keep their electricity revenue, it becomes
difficult to measure whether the solar farm even required carbon incentives in
the first place. Many solar farms are highly profitable even without carbon
based incentives, and therefore allocating carbon credit money to them is a
waste.

Even if the solar farm can demonstrate that they would not be profitable
without carbon incentives, it is still difficult to validate that the solar
farm is receiving just enough carbon incentives.

By requiring solar farms to give up 100% of their electricity revenue to other
solar farms, Glow creates a competitive rewards system which favors solar farms
that are as cost-efficient as possible in producing carbon credits. Solar farms
receive cash rewards from other solar farms proportional to the number of
carbon credits that they produce, and solar farms receive token rewards from
the Glow protocol proportional to the amount of cash they contribute to other
solar farms.

The net effect of this incentive structure strongly favors solar farms that are
just a hair's breadth away from profitability. Solar farms that are profitable
without carbon incentives will prefer not to participate in Glow, because they
will want to keep all of their valuable electricity revenue. And solar farms
that produce very little revenue will not receive enough Glow tokens to remain
competitive.

In summary, the Glow Protocol is a blockchain native single methodology
pay-it-forward carbon credit standard focused on solar that uses emissionality
to measure marginal carbon impact and uses forward productivity and competition
to establish additionality.

**Comment by DavidVorick** [comment]
## What is Glow?

The Glow Protocol is a blockchain native single methodology pay-it-forward
carbon credit standard focused on solar that uses emissionality to measure
marginal carbon impact and uses forward productivity and competition to
establish additionality.

## What does that mean?

The Glow Protocol is a standard for certifying carbon credits. A carbon credit
is a metaphorical object that represents one ton of carbon dioxide that would
otherwise be in the atmosphere. There are many carbon credit standards today,
and some reputable examples include the American Carbon Registry and CORSIA.

The Glow Protocol appraises projects that reduce carbon dioxide emissions, and
issues one Glow Carbon Credit for each metric ton of carbon dioxide that is
successfully prevented from being released into the atmosphere. The Glow
Protocol takes responsibility for the accurate appraisal of emissions
reductions and uses its own reputation to vouch for the quality of the carbon
credits that are created by Glow projects.

Glow is a blockchain-native protocol, which means all of the governance and
carbon credit issuance happens on the Ethereum blockchain. Blockchain
technology is a good match for the carbon credit industry, as it can give
strong guarantees about who owns the carbon credits, how many carbon credits
are in existence, and the current state of each carbon credit.

Many previously existing blockchain based carbon credit platforms struggled
because it is difficult to migrate legacy carbon credits to the blockchain, and
as a result many platforms either had few carbon credits or carbon credits of
very low quality. Glow supports the rest of the carbon credit blockchain
ecosystem by providing a source of scalable, high quality carbon credits that
permissionlessly interact with the rest of the crypto ecosystem.

Furthermore, it is much easier to migrate a blockchain based carbon credit into
the legacy carbon credit ecosystem than it is to go the other way. Carbon
credit buyers that are not yet ready to make the leap into Blockchain have the
ability to buy and retire Glow carbon credits using traditional payment flows.

Most carbon credit standards operate by giving carbon credits to projects that
reduce emissions or sequester carbon. Carbon credit buyers then purchase the
credits, effectively paying retroactively for change that has already occurred
and receiving little to no assurance that their money will be used to
incentivize future change.

The Glow Protocol uses a pay-it-forward financing structure that ensures all
money spent on carbon credits gets used to drive future carbon credit
production. As carbon credits are sold, the money is used to fuel the incentive
pool that pays out to future carbon projects.

This means that Glow is structured differently than traditional carbon credit
standards. Projects on Glow directly receive cash and tokens for producing
carbon credits instead of receiving carbon credits. The carbon credits, once
produced, are sold by the Glow protocol to fund future projects.

The Glow Protocol is a single methodology carbon credit standard, which means
that Glow only certifies exactly one type of carbon credit. Specifically, the
Glow Protocol certifies carbon credits when newly constructed solar farms
displace dirty energy from modern power grids. Dirty energy makes up more than
30% of all global emissions, giving Glow the opportunity to scale to producing
tens of billions of carbon credits per year even though Glow only uses a single
methodology to produce carbon credits.

The choice to use a single methodology is intentional. The carbon credit
ecosystem today features over one hundred different methodologies, each with
their own controversies and criticisms. Most buyers of carbon credits have
little hope of understanding what exactly they are purchasing, and education is
difficult because every carbon credit has its own story, and its own drawbacks.

By focusing on a single methodology, the Glow protocol is better able to scale
up, better able to address shortcomings and criticisms, and is better able to
give carbon credit buyers confidence in the credits they are purchasing.

The Glow protocol measures carbon dioxide emissions reductions using a strategy
called emissionality. Emissionality uses the rich, regulated data feeds of
modern power grids to observe which power plants reduce their output in
response to new solar farms joining the power grid. This allows Glow to measure
the precise emissions impact of a single solar farm, and also means that each
solar farm will produce different amounts of carbon credits based on where they
are in the world.

Emissionality can only be measured on grids that publish high quality data
about which power plants are active and how many emissions are associated with
each power plant. Thankfully, modern regulations require this data to be
available, and thus Glow is able to certify solar in the entire continental US,
most of Europe, much of Asia, and significant portions of the rest of the
world, with more regions being supported every year. Glow sources most of its
data through a non-profit provider called WattTime.

The Glow protocol features a novel requirement called forward productivity,
which requires solar farms to contribute 100% of their gross electricity
revenue towards the incentive pool that rewards solar farms for producing
carbon credits.

Though forward productivity sounds unusual at first, it solves an important
incentive alignment problem that exists in much of the carbon credit space.
Carbon credit buyers are spending money because they want to change the world.
This means that their money needs to enable projects that would not exist
otherwise.

If solar farms are allowed to keep their electricity revenue, it becomes
difficult to measure whether the solar farm required carbon incentives at all.
Many solar farms are highly profitable even without carbon based incentives,
and allocating incentives to those farms is a waste.

Even if a solar farm can demonstrate that it would not be profitable without
carbon incentives, it can be difficult to determine the exact amount of
incentives that are required to make the project viable.

By requiring solar farms to give up 100% of their electricity revenue to other
solar farms, Glow creates a competitive rewards system which favors solar farms
that are as cost-efficient as possible in producing carbon credits. Solar farms
receive cash rewards from other solar farms proportional to the number of
carbon credits that they produce, and solar farms receive Glow token rewards
proportional to the amount of cash they contribute to other solar farms.

This incentive structure most strongly favors solar farms that are just a
hair's breadth away from profitability. Solar farms that are profitable without
incentives will not participate in Glow, because they will prefer to keep their
valuable electricity revenue. And solar farms that produce very little revenue
overall will be unable to participate, because they won't receive enough Glow
tokens to be competitive with higher revenue solar farms.

In summary, the Glow Protocol is a blockchain native single methodology
pay-it-forward carbon credit standard focused on solar that uses emissionality
to measure marginal carbon impact and uses forward productivity and competition
to establish additionality.
